### PR TITLE
Amp analytics: Documentation for SUM function

### PR DESCRIPTION
**Pull request by pomeroyr**
Documentation for #12213 .

### CODE REVIEW

**Comment by pomeroyr** [comment]
@zhouyx @dvoytenko @lannka

File: spec/amp-var-substitutions.md [hunk 1]
```diff
@@ -478,6 +479,17 @@ Use `COUNTER(name)` to generate an index for the given request. Counts start at
 * **platform variable**: `COUNTER`
 * **amp-analytics variable**: `${counter}`
 
+#### Sum
+
+Use `SUM(name, value)` to generate a sum for the given request. Values are added per a given `name`.
```
Comment (lannka) [inline spec/amp-var-substitutions.md]: is the space after `,` intended?

File: spec/amp-var-substitutions.md [hunk 2]
```diff
@@ -478,6 +479,17 @@ Use `COUNTER(name)` to generate an index for the given request. Counts start at
...
+
+* **platform variable**: `SUM`
+*  Example: <br>
+  ```html
+  <amp-pixel src="https://foo.com/pixel?SUM('pixel1', 5)"></amp-pixel>
```
Comment (lannka) [inline spec/amp-var-substitutions.md]: to follow the convention, there should not be the single quote `'` around string literals.

File: spec/amp-var-substitutions.md (hunk 1 above)
Comment (pomeroyr) [reply spec/amp-var-substitutions.md]: Done.

File: spec/amp-var-substitutions.md (hunk 2 above)
Comment (pomeroyr) [reply spec/amp-var-substitutions.md]: Done.

**Comment by pomeroyr** [comment]
Comments addressed. PTAL @lannka

**Pull request closed without merging**